.github/workflows/validate-pr.yml

```
@@ -0,0 +1,419 @@
+            echo ""
+            echo "🎉 All checks passed! PR is ready for review."
+            exit 0
+          else
+            echo ""
+            echo "❌ Some checks failed. Please review the issues above."
+            exit 1
+          fi
+          
+      - name: Comment on PR
+        if: always()
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)